src/phoenix/server/app.py

```diff
@@ -26,22 +28,60 @@
 
 logger = logging.getLogger(__name__)
 
+templates = Jinja2Templates(directory=SERVER_DIR / "templates")
+
+
+@dataclass(frozen=True)
+class Config:
```

Comment:
nit. maybe `PhoenixConfig`, like [`StrawberryConfig`](https://github.com/strawberry-graphql/strawberry/blob/9c946a2d4f56b81dfb0e58e4a9ed2aa89adfc1c8/strawberry/schema/config.py#L10) since we also have uvicorn [Config](https://github.com/encode/uvicorn/blob/d3cb170262b9da62db9b5a573b6abcee4e5f3fcc/uvicorn/config.py#L187)?

Reply:
I made it AppConfig